src/components/MovieCard.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Box,
+  CircularProgress,
+  CircularProgressLabel,
+  Grid,
+  Heading,
+  Image,
+  Text,
+  VStack,
+  GridItem,
+  Stack,
+  Flex,
+} from "@chakra-ui/react";
+
+type MovieCardProps =
+  | { isLoading: true }
+  | {
+      poster_path: string;
+      title: string;
+      release_date: string;
+      vote_count: number;
```

Comment:
di JS sebisa mungkin pake camelCase aja ya, jangan dicampur sama snake_case

```suggestion
      posterPath: string;
      title: string;
      releaseDate: string;
      voteCount: number;
```

Reply:
Meskipun nanti properti dari data bentukmya snake_case ?

Reply:
iya, kalau yang dari API gapapa snake_case, tapi di component dan tempat2 lain di FE, baiknya pake camelCase